code/src/components/Form.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react'
```

Comment:
So nicely structured and clear, very easy to follow what is going on and good names of the components!